lists/pk.csv

```diff
@@ -945,3 +945,4 @@ http://islamandhomosexuality.com/5-ways-pakistan-gay-friendly/,LGBT,LGBT,2016-07
 http://www.humanrightsfirst.org/blog/lgbt-voices-equality-pakistan/,LGBT,LGBT,2016-07-20,OONI,
 http://www.towleroad.com/2016/06/lgbt-pakistan/,LGBT,LGBT,2016-07-20,OONI,
 http://www.pakistaniwomen.org/,HUMR,Human Rights Issues,2016-07-20,OONI,
+http://www.nydailynews.com/,NEWS,News Media,2017-02-09,,
```

Comment:
You should add a source here. It can also be ANON if you don't want to leave anything.

Also I think it needs to be rebased onto master.